conda/recipes/legate-dataframe/meta.yaml

```diff
@@ -61,14 +61,14 @@ requirements:
     # is compatible with an existing cupynumeric version.
     # (may also stabilize not using debug/sanitizer builds)
     - cupynumeric
-    - cython>=3.0.0
     - python
-    - pip
```

Comment:
I think you still need `pip` as a host dependency, because the build ends up invoking it:

https://github.com/rapidsai/legate-dataframe/blob/dd9bc3ccd4a6e70d5ef8ac78ca525534a592f3d9/build.sh#L158

From the logs ([link](https://github.com/rapidsai/legate-dataframe/actions/runs/12178643194/job/33969113140?pr=8#step:5:266)) it looks like you're still getting it, but maybe only transitively as a dependency of something else. It'd be better to specify it explicitly, to prevent surprises from that changing in the future (as recently happened to some projects with `setuptools`, e.g. https://github.com/rapidsai/dependency-file-generator/pull/113#issuecomment-2402855030).